apps/server/src/shared/domain/entity/submission.entity.spec.ts

```diff
@@ -14,28 +14,41 @@ describe('Submission entity', () => {
 		await setupEntities();
 	});
 
-	it('should be graded if grade percentage is set', () => {
-		const submission = buildSubmission();
-		submission.grade = 50;
-		expect(submission.isGraded()).toEqual(true);
-	});
+	describe('isGraded', () => {
+		it('should be graded if grade percentage is set', () => {
+			const submission = buildSubmission();
```

Comment:
i think we do not need a `buildSubmission` method here. the factory already adds the user and the task (because both are required)
https://github.com/hpi-schul-cloud/schulcloud-server/blob/17d5bc5b1a3e63945f6775b8113e2791e1aa7217/apps/server/src/shared/testing/factory/submission.factory.ts#L14

Reply:
Then i removed the buildSubmission from all the tests.